defrag/modules/twitter.py

```diff
@@ -1,57 +1,68 @@
-from defrag.modules.helpers import Query
-from collections import namedtuple
-from defrag import app, TWITTER_CONSUMER_KEY, TWITTER_CONSUMER_SECRET, TWITTER_ACCESS_TOKEN, TWITTER_ACCESS_TOKEN_SECRET
+from datetime import datetime
+from pydantic.main import BaseModel
+from defrag.modules.helpers import CacheQuery, Query, QueryResponse
+from defrag import LOGGER, app, TWITTER_CONSUMER_KEY, TWITTER_CONSUMER_SECRET, TWITTER_ACCESS_TOKEN, TWITTER_ACCESS_TOKEN_SECRET
 from defrag.modules.helpers.cache_stores import CacheStrategy, QStore, RedisCacheStrategy
-from defrag.modules.helpers.data_manipulation import compose
 from defrag.modules.helpers.sync_utils import as_async
 from defrag.modules.helpers.services_manager import Run, ServiceTemplate, ServicesManager
 import twitter
-from typing import Any, List, NamedTuple, Optional
-from os import environ as env
+from typing import Any, Dict, List
 from operator import attrgetter
 
 """ INFO
 This module provides a class to
-serve as querying & storing point for recent tweets (sent by user "@openSUSE"). 
+serve as querying & storing point for recent tweets (sent by user "@openSUSE").
 Additional features could be:
 - posting (would require higher credentials)
-- fetching also Tweets *about* openSUSE 
+- fetching also Tweets *about* openSUSE
 to model data answering the question: "What are people talking about recently?"
 """
 
 __MOD_NAME__ = "twitter"
 
 
+api = twitter.Api(consumer_key=TWITTER_CONSUMER_KEY, consumer_secret=TWITTER_CONSUMER_SECRET,
```

Comment:
```
Traceback (most recent call last):
  File "/usr/lib64/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib64/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/jens/src/defrag-api/defrag/__main__.py", line 69, in <module>
    main()
  File "/home/jens/src/defrag-api/defrag/__main__.py", line 30, in main
    imported_module = importlib.import_module(
  File "/usr/lib64/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/jens/src/defrag-api/defrag/modules/twitter.py", line 24, in <module>
    api = twitter.Api(consumer_key=TWITTER_CONSUMER_KEY, consumer_secret=TWITTER_CONSUMER_SECRET,
AttributeError: module 'twitter' has no attribute 'Api'
```

Reply:
```shell
EBUG    defrag:redis.py:26 Opening pool...
[2021-08-19 12:12:55,242 | INFO] Registered: reddit
INFO     defrag:services_manager.py:110 Registered: reddit
PASSED
defrag/tests/unittests/test_reddit.py::test_reddit_handler PASSED
defrag/tests/unittests/test_reddit.py::test_reddit_search_handler PASSED

=================== warnings summary ===================
defrag/tests/conftest.py:5
  /home/hades/python/defrag/defrag/tests/conftest.py:5: PytestDeprecationWarning: @pytest.yield_fixture is deprecated.
  Use @pytest.fixture instead; they are the same.
    @pytest.yield_fixture(scope="session")

-- Docs: https://docs.pytest.org/en/stable/warnings.html
============= 4 passed, 1 warning in 4.00s =============
```
HOWEVER you are onto something because I don't see the twitter module in the requirements and apparently my virtual environment manager does NOT add it to the requirements. Will look into that.

Reply:
can you check after pulling?

Reply:
The error does not occur anymore. Still doesn't work for me as I currently do not have the required API key, but I trust that you have tested it.